Signal/src/call/Speakerbox/CallKitCallUIAdaptee.swift

```diff
@@ -201,20 +201,12 @@ final class CallKitCallUIAdaptee: NSObject, CallUIAdaptee, CXProviderDelegate {
         AssertIsOnMainThread()
         Logger.debug("\(self.TAG) \(#function)")
 
-        // TODO
-        // copied from Speakerbox, but is there a corallary with peerconnection, since peer connection starts the audio
-        // session when adding an audiotrack
-//        stopAudio()
+        // Stop any in-progress WebRTC related audio.
+        PeerConnectionClient.stopAudioSession()
```

Comment:
> why these static audio methods are on PCC.

Because it's interfacing with a WebRTC API. But I'm also not convinced that's a good reason, feel free to move them if there's a better spot.